caml-font.el

```diff
@@ -184,7 +184,7 @@
 ; match any sequence of non-special characters in a comment
 ; note: this is only to go faster than one character at a time
 (defconst caml-font-other-comment-re
-  "[^{(*\"'\012\015]+"
+  "[^A-Za-z_\300-\326\330-\366\370-\377{(*\"'\012\015]+"
```

Comment:
I do not understand: `A`–`Z` are not special, why are they excluded (`^`)?

Reply:
Otherwise in `(* f' '"' *)` the `f` will be matched as `caml-font-other-comment-re`, then `' '` as a character, and the rest as an unclosed string. With this change, `f'` is matched as `caml-font-ident-re` and `'"'` as a character.